setup.py

```diff
@@ -33,12 +33,11 @@
     keywords='SQLAlchemy collectd',
     author='Mike Bayer',
     author_email='mike@zzzcomputing.com',
-    url='http://github.org/sqlalchemy/sqlalchemy-collectd',
+    url='https://github.com/sqlalchemy/sqlalchemy-collectd',
     license='MIT',
     packages=find_packages(".", exclude=["*.tests"]),
     include_package_data=True,
     tests_require=['pytest', 'mock'],
-    setup_requires=['pytest-runner'],
```

Comment:
this is so you can run "python setup.py test".    if we take this out then we have to pick a different test runner which means we'd need to change "test=pytest" inside of setup.cfg and probably some other things.

Reply:
In a quick test, I failed to follow the goodpractices manual to by adding a additional class and to make it work. Maybe you'd have the time to look at it?